ui/main.qml

```diff
@@ -443,6 +450,15 @@ StatusWindow {
 
             onboardingStore: onboardingStore
 
+            fetchSecretViaBiometrics: function (profileId, callback) {
+                const isKeycardProfile = SQUtils.ModelUtils.getByKey(
+                                           onboardingStore.loginAccountsModel, "keyUid",
+                                           profileId, "keycardCreatedAccount")
+
+                const reason = isKeycardProfile ? qsTr("fetch pin") : qsTr("fetch password")
```

Comment:
Is the reason displayed in the UI somewhere?

Reply:
![image](https://github.com/user-attachments/assets/7a913a26-0941-4b91-a947-6d9f4f6eaad7)

That's it.